### ISSUE TITLE
URLSegment custom right label takes up too much space

### ISSUE THREAD

**Issue by sunnysideup** [silverstripe/silverstripe-framework#8706]
## Affected Version

SS4.3

## Description

![image](https://user-images.githubusercontent.com/167154/50876742-7af0cd80-1433-11e9-8f0c-bfc348dc5d53.png)

As you can see above, the UI needs fixing here. Not sure what is the best way, but why not put the right label underneath the field?

**Comment by maxime-rainville** [comment]
I don't seem to be getting this warning at all. Is that custom message you added?

**Comment by sunnysideup** [comment]
Ahh, yes, sorry, I did not realise - that is a custom message.  However, the right field label does seem to be allocated too little space on a laptop size screen:

Here is what it looks like on a giant screen:

![image](https://user-images.githubusercontent.com/167154/50879871-526fd000-1441-11e9-9230-9cac73532cd4.png)

and this is what it looks like on a small screen:

![image](https://user-images.githubusercontent.com/167154/50879887-61ef1900-1441-11e9-9fad-87c8bbf924f0.png)

and a tiny screen:
![image](https://user-images.githubusercontent.com/167154/50879907-79c69d00-1441-11e9-976d-6158d61c25bb.png)


I would suggest that more screen sizes do as the tiny screen as this allows for a lot more content to be stuffed into the right label, which is exactly what it is for: additional information.

**Comment by robbieaverill** [comment]
cc @clarkepaul

**Comment by clarkepaul** [comment]
You are right that the positioning of the label got messed up a while back so isn't styled correctly, there are also more styling issues of the right label when you look at the page metadata accordion area.

If it's possible for the URL Segment field try to use the form__field-description instead of the right label, as it sits below the field in all cases. When the label on the right I think it should be slightly closer to the field, and at extremely large sizes we could give more space to the right label but there will be a certain screen size when it would be much better.
